platform/qt/src/qmapboxgl.cpp

```diff
@@ -118,6 +118,38 @@ auto fromQStringList(const QStringList &list)
 
 }
 
+/*!
+    \class QMapboxGLSettings
+    \brief The QMapboxGLSettings stores the initial configuration for a QMapboxGL.
```

Comment:
`The QMapboxGLSettings` 👉 `The QMapboxGLSettings class`
`for a QMapboxGL` 👉 `for QMapboxGL object`.